python_modules/libraries/dagster-databricks/dagster_databricks/databricks_pyspark_step_launcher.py

```diff
@@ -197,3 +216,69 @@ def _internal_dbfs_path(self, run_id, step_key, filename):
         """Scripts running on Databricks should access DBFS at /dbfs/."""
         path = "/".join([self.staging_prefix, run_id, step_key, os.path.basename(filename)])
         return "/dbfs/{}".format(path)
+
+
+class DatabricksConfig:
+    def __init__(self, storage, secrets):
```

Comment:
Would you mind including some docstring here explaining the purpose of this object and why we don't use the "regular" Dagster config system for it?

Reply:
I've added one now, hopefully it addresses everything.